index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+  </head>
+
+  <body>
+    <nav>
+      <a class="logo" href="/"
+        ><img
+          src="./image/Img_logo.png"
+          alt="판다마켓 로고 이미지"
+          width="153px"
+          height="51px"
+      /></a>
+      <a class="login-btn" href="/login">로그인</a>
+    </nav>
+
+    <header>
+      <div class="header-content">
+        <h1 class="title">일상의 모든 물건을<br />거래해 보세요</h1>
+        <a class="header-btn" href="/items">구경하러 가기</a>
+      </div>
+      <img
+        class="header-img"
+        src="./image/Img_home_top.png"
+        alt="판다가 손을 흔드는 이미지"
+        width="996px"
+        height="447px"
+      />
```

Comment:
```suggestion
      <img
        class="header-img"
        src="./image/Img_home_top.png"
        alt="판다가 손을 흔드는 이미지"
        width="996px"
        height="447px"
      />
```

img 태그에 width, height 속성을 설정하는것과 css로 width, height를 설정하는건 둘다 장단점이 있습니다.

* img 태그에 width,height를 설정하면 브라우저가 이미지 크기를 미리 알 수 있어서 빠른 랜더링이 가능하지만,
width, height의 원래 비율과 상관없이 지정 한 크기로 그려집니다.

* css 속성은, 미디어 쿼리로 반응형으로 다루거나, 다양한 단위로 표현할 수 있습니다. (`%`,`em`,`rem`등등) 그리고 width, 혹은 height의 값을 하나만 설정하면 비율에 맞게 계산됩니다. 하지만 브라우저는 이미지 크기를 몰라서 html이 먼저 그려지고, 이미지가 껴지면 layout의 이동이 생기거나 초기 렌더링이 조금 지연될 수 있습니다(속도는 프로젝트가 커지고,복잡하면 그때 고민해도됩니다.)

상황에 맞게 선택하시면 좋을것같아요 :-)